tools/cell_census_builder/tests/test_builder.py

```diff
@@ -87,13 +91,13 @@ def h5ad(self) -> anndata.AnnData:
         obs = obs_dataframe
 
         # Create vars
-        feature_name = pd.Series(data=["a", "b", "c", "d"])
+        feature_name = pd.Series(data=[f"{organism.name}_{g}" for g in ["a", "b", "c", "d"]])
         var_dataframe = pd.DataFrame(
```

Comment:
while it works today, if we ever start enforcing the non-uniqueness requirement for feature_reference, this dataframe will fail as every row is identical (except for the index).

Easy to fix - just generate the `data` rows using same method as `feature_name`

Reply:
sorry, what is the "the non-uniqueness requirement for feature_reference"?

Reply:
I was just pointing out that all of the dataframe rows are identical, and that might not be the best test.  This is a nit - just ignore it.